docs/proposals/PROPOSAL_C.md

```diff
@@ -321,3 +267,66 @@ Gives response:
+
+#### Backwards Compatibility
+
+1. As a user, I want to be sure that existing container runtimes are not affected by any other type of registry artifact.
+  - Yes: Existing manifests such as container or index manifests are not affected.
+1. As a user, I want to move container images to and from registries that do not support reference types.
+  - Partial: Registries that do not support node manifests require the client to create tags for each of the embedded artifacts before pushing them. All the information in description and reference will be lost. Registries that support node manifests have no effect on artifacts that require tagging.
+1. As an artifact producer, I want to tag artifacts that I can pull by said tag, even if they contain references to other artifacts.
+  - Yes: Node manifests will not affect tags on other artifacts.
+1. As an artifact producer, I want be sure that pushing an artifact to a repository will not affect a copy of the same artifact previously created and referenced by a manifest list existing in another repository on the same registry.
+  - Yes: Node manifests do not modify existing artifacts nor do they create duplicates of the same artifact, as is the case with existing registries.
+1. As a tool writer, I want to identify whether a registry supports reference types or not.
+  - Yes: use `GET /v3/`. Catalog listing is not prescribed in this proposal.
+1. As a tool writer, I would like the option to perform a server side blob mount when copying a large artifact between repositories.
+  - Yes: If a large blob is pushed to the registry and then referenced by a Node manifest, a server side mount should be possible.
+1. As a tool writer, I want to be able to include reference types within the Image Layout filesystem format.
+  - Yes: The existing image layout will be unaffected.
+
+#### Content Management
+
+1. As a user, I want to store one or more artifacts in a registry with a reference to another related artifact.
+  - Yes: Node manifests can group one or more artifacts with a reference to another manifest.
+1. As a user, when I delete an artifact, I want the option to delete one or more artifacts referencing it.
+  - Partial: use `GET /v3/<name>/referrers/<reference>` to get a list of referring artifacts, use the client to delete specific ones, then push a new Node manifest containing the updated object list. To delete the whole manifest, use `DELETE /v3/<name>/manifests/<reference>`.
+1. As a user, I want to push an artifact that references another artifact that doesn't exist on the registry yet.
+  - Yes: This is should be possible if registries don't perform a check that the digest exists.
+1. As a user, I want to move an artifact and one or more artifacts referencing it from one registry to another.
+  - Partial: The client is responsible for selection of the artifacts and pushing to the target registries, including checks if the registry supports node manifests.
```

Comment:
The only question mark for me is gracefully handling backwards compatibility that doesn't require pulling every reference on every sync. Otherwise I'd call this a yes, even though it requires client tooling, because there's no server side tooling in OCI for copying between registries today.